### PR TITLE
Badge component

### PR DESCRIPTION
**Pull request by atul-gairola**

### CODE REVIEW

File: styles/components/badge.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  width: fit-content;
+}
+
+.badge {
+  border-radius: 50%;
+  border: 2px solid var(--white);
+  padding: 5px;
+  background: var(--info);
+  position: absolute;
+  top: 0px;
```
Comment (b0llu) [inline styles/components/badge.css]: you can just write 0 no need to add px next to it

File: styles/components/badge.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+.badge-yellow {
+  background: var(--warning);
+}
+
+.badge-text {
+  top: 0px;
+  right: 0px;
+  font-weight: 500;
+  font-size: 0.4rem;
```
Comment (b0llu) [inline styles/components/badge.css:39]: if the size is less than 1rem use px

File: styles/documentation.css [hunk 3]
```diff
@@ -110,3 +121,19 @@ header {
...
+  width: 100%;
+}
+
+.icon-badges-container {
+  display: flex;
+  align-items: center;
+  gap: 20px;
+  margin-bottom: 20px;
+  justify-content: center;
+}
```
Comment (b0llu) [inline styles/documentation.css:139]: either use rems for all or use px. i would recommend rem as its responsive

**Review by b0llu — APPROVED** [review]

File: styles/components/badge.css (hunk 2 above)
Comment (mikesch803) [inline styles/components/badge.css:39]: you should increase the font size as it looks tiny in the preview

**Review by mikesch803 — CHANGES_REQUESTED** [review]
i put some suggestion here, and also in previous pr, please look into it and, your iframe are empty in preview
image

**Review by aryanandarnav13 — APPROVED** [review]

**Pull request merged**